crate/guides/0.7.0/todomvc_routing.md

```diff
@@ -11,7 +11,7 @@ Let's implement proper filter handling in our `view` functions. (We could do it
 ```rust
 fn view(model: &Model) -> Vec<Node<Msg>> {
     ...
-            view_footer(&model.todos, model.filter),
+            view_main(&model.todos,model.selected_todo.as_ref(), model.filter),
```

Comment:
I'm surprised that formatter didn't add a space between `,` and `model.se..` => `&model.todos,model.selected_todo.as_ref()`.

Reply:
Please just add a space and maybe try to find out why the formatter hasn't fixed it. And I'll merge it.

Reply:
Formatter doesn't do markdown files, right? I'll put the space in.

Reply:
Yeah you are right, I was thinking about the code in the repo, sorry for confusion. It looks like I copied the code before formatting, my bad.